lib/stages/rawFileRead.cpp

```diff
@@ -67,8 +67,13 @@ void rawFileRead::main_thread() {
             snprintf(full_path, full_path_len, "%s/%s_%s_%07d.%s", base_dir.c_str(), hostname,
                      file_name.c_str(), file_num, file_ext.c_str());
         } else {
-            snprintf(full_path, full_path_len, "%s/%s_%07d.%s", base_dir.c_str(), file_name.c_str(),
-                     file_num, file_ext.c_str());
+            if (ndisk_wrote)
```

Comment:
Why don't you loop over the num_disks?  If you are reading nDisk written files, don't you want to read from each disk?   I would also change this variable name to `ndisk_mode`.

Reply:
This function was designed for different name conventions, during the development. I did not mean to change this stage. But somehow I have to commit it. Do you want me to change the stage?